scripts/linters/pylint_extensions.py

```diff
@@ -589,12 +589,12 @@ class DocstringParameterChecker(checkers.BaseChecker):
     DOCSTRING_SECTION_YIELDS = 'yields'
     DOCSTRING_SECTION_RAISES = 'raises'
 
-    def visit_classdef(self, node):
```

Comment:
Do you want to say it is written for module node rather than Class node? I don't think so.